notebooks/biodiversity_mapping.ipynb

```diff
@@ -0,0 +1,2552 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    df = occ.search(taxonid=1363, startdepth=STARTDEPTH, enddepth=ENDDEPTH)
</pre><p>Suggest adding a comment here :</p><p> </p><p> # <a href="https://www.marinespecies.org/aphia.php?p=taxdetails&amp;id=1363" target="_blank">Scleractinia</a> == taxon 1363 </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Thank you so much for the review! I'll update the comments.